vimiv/api/_modules.py

```diff
@@ -89,6 +89,54 @@ def copy_name(abspath: bool = False, primary: bool = False) -> None:
     clipboard.setText(name, mode=mode)
 
 
+@api.keybindings.register("yi", "copy-image")
+@api.keybindings.register("yI", "copy-image --primary")
+@api.commands.register()
+def copy_image(
+    primary: bool = False, width: int = -1, height: int = -1, size: int = -1
```

Comment:
I would personally prefer to use `None` instead of `-1` as it fits better to the `Optional` character of this flag.

Another random idea I had is to support `count` here which would set the size using `count` so e.g. `800yi` would result in the same behaviour as `:copy-image --size 800` to allow setting the size without interacting with the command line.

Reply:
Imrpoved the None in cdbacf4

Added the count support in d767e5b55d45f14ba22f5cde86ad00f337d3c4a4. Have I added the documentation for the count correctly to the docstring?

Reply:
Looks good to me except for possibly a missing whitespace after `--size`.

Reply:
good catch, fixed in 421e0c65c372d0672455c22bcfd9a32d602e16e4